openmdao/core/options.py

```diff
@@ -90,6 +89,15 @@ def get(self, name, default=None):
             return self._options[name]['val']
         return default
 
+    def update(self, options):
+        self._options.update(deepcopy(options._options))
```

Comment:
Function needs a docstring.  Otherwise, this is a pretty lightweight dict, so I wonder if it really needs this function.